.github/workflows/semantic_release.yml

```diff
@@ -19,19 +19,10 @@ jobs:
           node-version: "14"
       - name: Setup node deps
         run: npm i conventional-changelog-conventionalcommits@4 -D
-      # - name: Setup docs node deps
-      #   run: |
-      #     cd docs
-      #     npm i
-      #     npm run build
       - name: Compress tar.gz
-        run: |
-          tar czvf lenra-api.tar.gz api/
-      # tar czvf lenra-api-docs.tar.gz docs/build/
+        run: tar czvf lenra-api.tar.gz api/
       - name: Compress zip
-        run: |
-          zip -r lenra-api.zip api/
-      # zip -r lenra-api-docs.zip docs/build/
+        run: zip -r lenra-api-docs.zip docs/build/
```

Comment:
this one changed from lenra-api to lenta-api-docs.
Is this intended ?

Reply:
oops my bad